SlevomatCodingStandard/Sniffs/Classes/TraitUseSpacingSniff.php

```diff
@@ -69,16 +72,19 @@ public function process(File $phpcsFile, $classPointer): void
 
 	private function checkLinesBeforeFirstUse(File $phpcsFile, int $firstUsePointer): void
 	{
+		$tokens = $phpcsFile->getTokens();
+
 		/** @var int $pointerBeforeFirstUse */
 		$pointerBeforeFirstUse = TokenHelper::findPreviousExcluding($phpcsFile, T_WHITESPACE, $firstUsePointer - 1);
+		$isAtTheStartOfClass = $tokens[$pointerBeforeFirstUse]['code'] === T_OPEN_CURLY_BRACKET;
```

Comment:
It does not work for code like this:

```php
Class Anything 
{
	
	public function whatever()
	{
	}

	use SomeTrait;
}
```

Reply:
I cannot reproduce an error with your example.
If I do not change the default settings of the sniff, it just throws the error that a blank line is expected after the `use`. Which is correct.
What exactly is not working in your example?